src/audio/kpb.c

```diff
@@ -60,11 +60,21 @@ static struct comp_dev *kpb_new(struct sof_ipc_comp *comp)
 {
 	struct comp_dev *dev;
 	struct sof_ipc_comp_kpb *kpb;
-	struct sof_ipc_comp_kpb *ipc_kpb = (struct sof_ipc_comp_kpb *)comp;
+//	struct sof_ipc_comp_kpb *ipc_kpb = (struct sof_ipc_comp_kpb *)comp;
+	struct sof_ipc_comp_kpb kpb_ipc, *ipc_kpb;
```

Comment:
Even if it is a temporary fix if you want this to get merged please just remove the comment.